sql/project_record_ids.sql

```diff
@@ -22,11 +22,35 @@ OUTPUTS:
 
 -- Copy pre-DOB match _project_record_ids into project_record_ids;
 DROP TABLE IF EXISTS project_record_ids;
+DROP TABLE IF EXISTS verified_matches;
```

Comment:
This is fine to keep in, but `verified_matches` is only a temporary table created with a `WITH` statement so there should never be a table that needs dropping.